apps/docs/.browserslistrc

```diff
@@ -5,8 +5,4 @@
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
```

Comment:
@Karnaukhov-kh will this be a problem for our docs page?

Reply:
The docs page is not an Angular app.